lib/WGmeta/Cli/Commands/Show.pm

```diff
@@ -0,0 +1,247 @@
+package WGmeta::Cli::Commands::Show;
+use strict;
+use warnings FATAL => 'all';
+use experimental 'signatures';
+use feature 'switch';
```

Comment:
do you actually use given/when ? the whole statement has been surrounded by contraversy for some time, dues to the smart match problems ... so I have been avoiding it, using the for () { structure instead ... but since it is still around it is probably save to use now ...

Reply:
This is actually a leftover from a different implementation...it's not needed anymore and therefore we can safely remove it :)